manim_slides/convert.py

```diff
@@ -742,6 +771,17 @@ def convert(
         else:
             cls = Converter.from_string(to)
 
+        # Change data_uri to one_file and print a warning
+        if "data_uri" in config_options:
+            warnings.simplefilter("default")
+            warnings.warn(
+                "The 'data_uri' configuration option is deprecated. "
+                "Use 'one_file' instead.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+            config_options["one_file"] = config_options.pop("data_uri")
```

Comment:
I would go for this instead
```suggestion
            config_options.setdefaut("one_file", config_options.pop("data_uri"))
```
so we avoid overwriting any existing value.

Reply:
I committed your suggestion, but the pre-commit check is not happy about it: https://results.pre-commit.ci/run/github/513032928/1735832808.GEGqiHM6SuCEU4ZpaxlliA